page/javascript-101/types.md

```diff
@@ -66,7 +66,7 @@ var bar2;
 
 ## Objects
 
-Everything else in JavaScript is considered an object. While there are [numerous built-in objects](https://developer.mozilla.org/en/JavaScript/Reference#Global_Objects, "MDN - Global Object Reference"), this chapter will cover:
+Everything else in JavaScript is considered an object. While there are [numerous built-in objects](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects "MDN - Standard built-in objects"), this chapter will cover:
```

Comment:
I'm aware of that, but the title of the new page is "Standard built-in objects". If you think the old link is better, I should also change the link to [the right version of the old one](https://developer.mozilla.org/docs/Web/JavaScript/Reference#Global_Objects) (the old link without the extra comma and with language redirection) for consistency. WDYT?